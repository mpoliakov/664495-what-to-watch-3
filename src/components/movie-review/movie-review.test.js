import React from 'react';
import renderer from 'react-test-renderer';
import MovieReview from './movie-review.jsx';

it(`<MovieReview/> is rendered correctly`, () => {
  const review = {
    id: `2fc32fa3-e5d4-4e7d-8f58-397cc8c9bba7`,
    text: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
    author: `Kate Muir`,
    date: `2016-12-24`,
    rating: 8.9
  };

  const component = renderer.create(<MovieReview review={review}/>).toJSON();
  expect(component).toMatchSnapshot();
});
