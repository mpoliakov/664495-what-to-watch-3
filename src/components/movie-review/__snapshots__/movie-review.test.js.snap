// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieReview/> is rendered correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      Poised to be an instant classic, almost everything about this film is phenomenal - the acting, the cinematography, the discography, etc.
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Isaac
      </cite>
      <time
        className="review__date"
        dateTime="2020-03-06T05:40:57.360Z"
      >
        March 6, 2020
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    2.6
  </div>
</div>
`;
